src/api/app/policies/project_policy.rb

```diff
@@ -1,17 +1,17 @@
 class ProjectPolicy < ApplicationPolicy
   def initialize(user, record)
-    raise Pundit::NotAuthorizedError, 'record does not exist' unless record
+    require_record(record)
```

Comment:
I think I'd prefer a call to super with flags if *not* required.

Reply:
how? :thinking: